### PR TITLE
Cycle 2408 Changelog

### LINKED ISSUE #123: Cycle 2408 Changelog

### PR DESCRIPTION
**Pull request by mwr2410**
Changelog for Cycle 2408

### CODE REVIEW

File: docs/changelog/2024/2408.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+- RNP X (AR): Waypoint `RIDGE` replaced by `ATRAB`
+
+-----
+
+### Standard Routes
+
+No changes have been made this cycle.
+
+### Other Dataset Changes
```
Comment (mwr2410) [inline docs/changelog/2024/2408.md:72]: May need to add items here, I don't think I missed anything?

File: docs/changelog/2024/2408.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+-----
+
+## Data Changes
+
+### Aerodrome
+
+#### NZCH: Christchurch
+
+- The following STARs have been revised:
+    - `BELEE4E`: MSA revised between `VANDA` and `BELEE`
```
Comment (rb-vatnz) [inline docs/changelog/2024/2408.md]:
```suggestion
    - `BELEE 4E`: MSA revised between `VANDA` and `BELEE`
```

File: docs/changelog/2024/2408.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+- VOR/DME RWY 34 has been amended with a new hold at `SANON`
+
+#### NZNS: Nelson
+
+- The following waypoints have been renamed:
+    - `QUAIL` is now `ENKIN`
+
+- The following STARs have been amended and re-sequenced:
+    - `LUBSI3C`,`BISEB3A`
```
Comment (rb-vatnz) [inline docs/changelog/2024/2408.md]:
```suggestion
    - `LUBSI 3C`
    - `BISEB 3A`
```

File: docs/changelog/2024/2408.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+    - RNP Y RWY 02: IAF `QUAIL` replaced by `ENKIN`
+
+#### NZQN: Queenstown
+
+- The following waypoints have been renamed:
+    - `BIKOS` is now `RIBDU`
+
+##### RWY 05
+
+- The following SIDs have been revised and re-sequenced:
```
Comment (rb-vatnz) [inline docs/changelog/2024/2408.md]:
```suggestion
- The following SIDs have been revised and re-sequenced with `BIKOS` replaced by `RIBDU`: 
```

File: docs/changelog/2024/2408.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+#### NZQN: Queenstown
+
+- The following waypoints have been renamed:
+    - `BIKOS` is now `RIBDU`
+
+##### RWY 05
+
+- The following SIDs have been revised and re-sequenced: 
+    - `IPNOR5A`,`GIXEL 2`: `BIKOS` replaced by `RIBDU`
```
Comment (rb-vatnz) [inline docs/changelog/2024/2408.md]:
```suggestion
    - `IPNOR 5A`
    - `GIXEL 2`
```

File: docs/changelog/2024/2408.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+
+- The following approaches have been revised:
+    - RNP Z (AR): Updated minima
+
+##### RWY 23
+
+- The following SIDs have been revised and re-sequenced: 
+    - `ANPOV5B`,`IPNOR5B`: Climb gradients reviewed
```
Comment (rb-vatnz) [inline docs/changelog/2024/2408.md]:
```suggestion
    - `ANPOV 5B` : `BIKOS` replaced by `RIBDU`, climb gradients reviewed
    - `IPNOR 5B` : `BIKOS` replaced by `RIBDU`, climb gradients reviewed
```

File: docs/changelog/2024/2408.md [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+    - `IPNOR5A`,`GIXEL 2`: `BIKOS` replaced by `RIBDU`
+
+- The following approaches have been revised:
+    - RNP Z (AR): Updated minima
+
+##### RWY 23
+
+- The following SIDs have been revised and re-sequenced: 
+    - `ANPOV5B`,`IPNOR5B`: Climb gradients reviewed 
+    - `REDOL3B`,`VALPI 3`
```
Comment (rb-vatnz) [inline docs/changelog/2024/2408.md]:
```suggestion
    - `REDOL 3B`
    - `VALPI 3`
```

File: docs/changelog/2024/2408.md [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+- The following approaches have been revised:
+    - RNP Z (AR): Updated minima
+
+#### NZTG: Tauranga
+
+- The follwing waypoints have been renamed:
+    - `MEZZO` is now `UKMIT`
+
+- The following STARs have been revised and re-sequenced:
+    - `OLDON1D`: Waypoint `MEZZO` replaced by `UKMIT`
```
Comment (rb-vatnz) [inline docs/changelog/2024/2408.md]:
```suggestion
    - `OLDON 1D`: Waypoint `MEZZO` replaced by `UKMIT` (Previously `OLDON 2A`)
```

File: docs/changelog/2024/2408.md [hunk 9]
```diff
@@ -0,0 +1,127 @@
...
+    - `OROPI2B`: Waypoint `SALOL` added to STAR.
```
Comment (rb-vatnz) [inline docs/changelog/2024/2408.md]:
```suggestion
    - `OROPI 2B`: Waypoint `SALOL` added to STAR.
```

File: docs/changelog/2024/2408.md [hunk 10]
```diff
@@ -0,0 +1,127 @@
...
+
+#### NZFA: Te Hapua Heliport
+
+- The following procedures have been created:
+    - `RNP 276`: **CAT H ONLY** approach with IAFs at `VETIR`,`OLTOD`,`KARNI`
+    - `KARNI1P`: new RNP SID, terminates at `KABIL`
```
Comment (rb-vatnz) [inline docs/changelog/2024/2408.md]:
```suggestion
    - `KARNI 1P`: new RNP SID, terminates at `KABIL`
```

File: docs/changelog/2024/2408.md [hunk 11]
```diff
@@ -0,0 +1,127 @@
...
+#### NZWN: Wellington
+
+- The following waypoints have been renamed:
+    - `RIDGE` is now `ATRAB`
+
+- The following STARs have been revised and re-sequenced:
+  
+##### RWY 16
+
+- `TPAPA3D`,`SIMZI3D`,`AVPOD3D`: now CAT `B`,`C`,`D`
```
Comment (rb-vatnz) [inline docs/changelog/2024/2408.md]:
```suggestion
- The following STARs have been reviewed and are now CAT `B`,`C`,`D`
   - `TPAPA 3D`
   - `SIMZI 3D`
   - `AVPOD 3D`
   - `WARDS 4D`: MSA between `PHNIX` and `DORBU` also revised
```

File: docs/changelog/2024/2408.md [hunk 12]
```diff
@@ -0,0 +1,127 @@
...
+
+- `WARDS4D`: now CAT `B`,`C`,`D`, MSA between `PHNIX` and `DORBU` revised
```
Comment (rb-vatnz) [inline docs/changelog/2024/2408.md]:
```suggestion
```

File: docs/changelog/2024/2408.md [hunk 13]
```diff
@@ -0,0 +1,127 @@
...
+- The following procedures have been created:
+    - `RNP 276`: **CAT H ONLY** approach with IAFs at `VETIR`,`OLTOD`,`KARNI`
+    - `KARNI1P`: new RNP SID, terminates at `KABIL`
+
+#### NZWN: Wellington
+
+- The following waypoints have been renamed:
+    - `RIDGE` is now `ATRAB`
+
+- The following STARs have been revised and re-sequenced:
```
Comment (rb-vatnz) [inline docs/changelog/2024/2408.md]:
```suggestion
```

File: docs/changelog/2024/2408.md [hunk 14]
```diff
@@ -0,0 +1,127 @@
...
+
+- `WARDS4D`: now CAT `B`,`C`,`D`, MSA between `PHNIX` and `DORBU` revised
+
+##### RWY 34
+
+- `SIMZI8A`: Track and MSA between `KRAKA` and `MEEDA` revised
+
+- `TPAPA5A`: Track between `BRUDA` and `PEGPU` revised
+    
+- `TPAPA3C`,`SIMZI4C`,`SULRI4C`,`WARDS4C`: now CAT `B`,`C`,`D` and the MSA between `KRAKA` and `MEEDA` revised.
```
Comment (rb-vatnz) [inline docs/changelog/2024/2408.md]:
```suggestion
- The following STARs have been reviewed and are now CAT `B`,`C`,`D`
   - `TPAPA 3C`
   - `SIMZI 4C`
   - `SULRI 4C`
   - `WARDS 4C`: MSA between `KRAKA` and `MEEDA` also revised
```

File: docs/changelog/2024/2408.md [hunk 15]
```diff
@@ -0,0 +1,127 @@
...
+  
+##### RWY 16
+
+- `TPAPA3D`,`SIMZI3D`,`AVPOD3D`: now CAT `B`,`C`,`D`
+
+- `WARDS4D`: now CAT `B`,`C`,`D`, MSA between `PHNIX` and `DORBU` revised
+
+##### RWY 34
+
+- `SIMZI8A`: Track and MSA between `KRAKA` and `MEEDA` revised
```
Comment (rb-vatnz) [inline docs/changelog/2024/2408.md]:
```suggestion
- The following STARs have been revised:
    - `SIMZI 8A`: Track and MSA between `KRAKA` and `MEEDA` revised
    - `TPAPA 5A`: Track between `BRUDA` and `PEGPU` revised
```

File: docs/changelog/2024/2408.md [hunk 16]
```diff
@@ -0,0 +1,127 @@
...
+
+- `TPAPA5A`: Track between `BRUDA` and `PEGPU` revised
```
Comment (rb-vatnz) [inline docs/changelog/2024/2408.md]:
```suggestion
```

Comment (mwr2410) [reply docs/changelog/2024/2408.md]: Nice suggestions, @rbywater  Cheers for those, i'll commit them shortly

**Review by mwr2410 — COMMENTED** [review]
Committed Richard's changes

**Review by dirtyformal — APPROVED** [review]

**Pull request merged**